content/docs/get-started/aws/create-project.md

```diff
@@ -40,6 +40,10 @@ $ pulumi new aws-python
 <div class="language-prologue-go"></div>
 
 ```bash
+# install the pulumi aws plugin
+# check for the release version here https://github.com/pulumi/pulumi-aws/releases
+$ pulumi plugin install resource aws 1.22.0
```

Comment:
Bummer that we need to hard-code this. Does `pulumi plugin install` automatically pick the latest is a version is unspecified? Or is there any way to inject the version using a template, similar to how the `{{< latest-version >}}` shortcode works for the CLI/SDK version?

Reply:
I agree this is a subpar experience. Two reasons I think this is ok for now:

1. This is strictly better than what we have today, which is a dead end.
2. @pgavlin has a P1 scheduled for m32 that will make this obsolete https://github.com/pulumi/pulumi/issues/1549. So this should be a temporary change. 

With that said I think we should just ship this now and clean it up as a part of go plugin discovery.

Reply:
This is fine -- but _please_ let's make sure to update the docs when we do this!!

Reply:
💯 I've added a comment into the issue to make sure this doesn't get lost. 

https://github.com/pulumi/pulumi/issues/1549#issuecomment-588607084